lnrpc/routerrpc/router.proto

```diff
@@ -500,11 +500,21 @@ service Router {
     rpc EstimateRouteFee (RouteFeeRequest) returns (RouteFeeResponse);
 
     /**
-    SendToRoute attempts to make a payment via the specified route. This method
+    Deprecated, use SendToRouteV2. SendToRoute attempts to make a payment via
+    the specified route. This method differs from SendPayment in that it
+    allows users to specify a full route manually. This can be used for
+    things like rebalancing, and atomic swaps.
+    */
+    rpc SendToRoute (SendToRouteRequest) returns (SendToRouteResponse) {
+        option deprecated = true;
+    }
+
+    /**
+    SendToRouteV2 attempts to make a payment via the specified route. This method
     differs from SendPayment in that it allows users to specify a full route
     manually. This can be used for things like rebalancing, and atomic swaps.
     */
-    rpc SendToRoute (SendToRouteRequest) returns (SendToRouteResponse);
+    rpc SendToRouteV2 (SendToRouteRequest) returns (lnrpc.HTLCAttempt);
```

Comment:
Perhaps mention here in the comment (which'll go in the rendered API docs), that this improves on the first version of `SendToRoute` by also returning full HTLC information.

Reply:
Added comment to the deprecated call. So that if we remove it, there won't be a reference to it in V2.